### PR TITLE
Groundstation api

### PR DESCRIPTION
**Pull request by ericboucher**

### CODE REVIEW

File: src/config/layers.json [hunk 1]
```diff
@@ -329,7 +329,8 @@
   "snow_groundstations": {
     "title": "Temperature (example data for 1-11 January 2020)",
     "type": "groundstation",
-    "data": "groundstationSnow",
+    "data": "http://119.40.103.16/temp/Temperature?beginDateTime=2020-03-01&endDateTime=2020-03-21",
```
Comment (mamadOuologuem) [inline src/config/layers.json:332]: Should `data` contains `beginDateTime` and `endDateTime` params? Or should them be set is the app with data selector

File: src/components/DataDrawer/Chart/index.tsx [hunk 2]
```diff
@@ -13,7 +13,7 @@ type ChartProps = {
 
 function colorShuffle(colors: string[]) {
   return colors.map((_, i) =>
-    (-1) ** i !== -1 ? colors[i] : colors[colors.length - i],
+    (-1) ** i !== -1 ? colors[i] : colors[colors.length - i - 1],
```
Comment (mamadOuologuem) [inline src/components/DataDrawer/Chart/index.tsx:16]: is this `(-1) ** i !== -1` use to check the parity of `i`?

I commonly saw `i % 2` for that (return `0` for even and `1` for odd)

File: src/config/layers.json (hunk 1 above)
Comment (ericboucher) [reply src/config/layers.json:332]: We'll keep it like this for now, if they want to improve it later, they can

File: src/components/DataDrawer/Chart/index.tsx (hunk 2 above)
Comment (ericboucher) [reply src/components/DataDrawer/Chart/index.tsx:16]: yup that's better

**Pull request merged**